dbms/src/Storages/KVStore/Read/ReadIndex.cpp

```diff
@@ -144,13 +142,17 @@ void WaitCheckRegionReadyImpl(
         max_wait_tick_time,
         get_wait_region_ready_timeout_sec);
 
-    std::unordered_set<RegionID> remain_regions;
-    std::unordered_map<RegionID, uint64_t> regions_to_check;
     Stopwatch region_check_watch;
+
+    std::unordered_set<RegionID> remain_regions;
     kvstore.traverseRegions(
         [&remain_regions](RegionID region_id, const RegionPtr &) { remain_regions.emplace(region_id); });
     const size_t total_regions_cnt = remain_regions.size();
 
+    // The regions fall behind Region leaders' commit index from TiKV. A map of {RegionID -> AppliedIndex}
+    std::unordered_map<RegionID, uint64_t> regions_to_check;
```

Comment:
Still `commit_index` here. I saw the value comes from `resp.read_index()` which is commit-index.

Reply:
Fixed